apps/vr-tests-react-components/src/stories/Combobox.stories.tsx

```diff
@@ -133,6 +133,16 @@ storiesOf('Combobox Converged', module)
       </Combobox>
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=31022&commitId=fb8458dde02857e45596657eaa5b01e1c903b6cd&env=prod&clientType=FLUENTUI)


  <details><summary><b>Combobox Converged</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Combobox Converged.When rendering inline, it should render on top of relatively positioned elements.active option.chromium.png|0| Added|
|Combobox Converged.When rendering inline, it should render on top of relatively positioned elements.default.chromium.png|0| Added|
|Combobox Converged.When rendering inline, it should render on top of relatively positioned elements.hover.chromium.png|0| Added|

</details>


  <details><summary><b>Dropdown Converged</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Dropdown Converged.When rendering inline, it should render on top of relatively positioned elements.active option.chromium.png|0| Added|
|Dropdown Converged.When rendering inline, it should render on top of relatively positioned elements.default.chromium.png|0| Added|
|Dropdown Converged.When rendering inline, it should render on top of relatively positioned elements.hover.chromium.png|0| Added|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>